exercises/colorsquare.js

```diff
@@ -14,7 +14,13 @@ class ColorSquare{
 		you'll need to bind the handleClick method to this object:
 			this.handleClick = this.handleClick.bind(this)
 	*/
-	constructor( ){
+	constructor( colorArray, colorIndex, className ){
+		this.colorArray = colorArray;
+		this.colorIndex = colorIndex;
+		this.className = className;
```

Comment:
while not needed, you can have a property called class, if you really wanted to.